console/war/src/main/java/org/frankframework/console/runner/ConsoleStandaloneInitializer.java

```diff
@@ -49,15 +51,26 @@ public void onStartup(ServletContext servletContext) throws ServletException {
 	// Optionally, in order to enable the ladybug the profile ladybug.file or ladybug.database can be enabled.
 	public static void main(String[] args) {
 		SpringApplication app = new SpringApplication();
-//		app.setAdditionalProfiles("ladybug.file");
-//		app.setAdditionalProfiles("ladybug.database");
 		app.setAllowBeanDefinitionOverriding(true);
 		app.setWebApplicationType(WebApplicationType.SERVLET);
+		app.addListeners(new FailedInitializationMonitor());
 		Set<String> set = new HashSet<>();
 		set.add("SpringBootContext.xml");
 		app.setSources(set);
 		app.addPrimarySources(List.of(WsSciWrapper.class));
+
 		app.run(args);
 	}
 
+	/**
```

Comment:
I really don't like this.
This means that you break all  applications deployed to this Tomcat, not just your own.

Generally this is considered a really bad practice.